src/Pages/adminEdit/AdminCreate.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react';
+import FoodMenu from './FoodMenu';
+import { INPUT_VALUES, CLOSED_DAY, CATEGORIES } from './adminEditData';
+import './AdminEdit.scss';
+
+const AdminCreate = () => {
+  const accesToken = localStorage.getItem('TOKEN');
+
+  const [input, setInput] = useState({
```

Comment:
```suggestion
  const [storeInfo, setStoreInfo] = useState({
```
가게들의 정보를 담는 state라서 이런 이름이 더 낫겠네요!